fx-domain/src/main/java/fx/domain/instruments.java

```diff
@@ -0,0 +1,5 @@
+package fx.domain;
+
+public class Instruments {
```

Comment:
not this way ( please google how to use enums or you can see java code of same standart classes